README.md

```diff
@@ -71,3 +71,21 @@ If any of these fail, this will be reported in the GitHub status checks
 interface.
 
 [travis]: https://travis-ci.org/alphagov/govuk-design-system
+
+## Automated Deployment
+We're using Netlify to automate our deployment.
```

Comment:
Do we need to make it clear that the main 'production' site is actually deployed to PaaS using Travis?

Reply:
Updated, added GOV.UK PaaS to automated checks and production deployment and refer to the Netlify as "Master deploy" now, instead of "Production deploy".